screens/product-listing-page/product-listing-page.html

```diff
@@ -252,5 +315,6 @@ <h5>PS4</h5>
 		<footer class="homepage-footer">
 			<p>Made with &hearts; by zoef shaikh</p>
 		</footer>
+		<script src="/app.js"></script>
```

Comment:
You can use script tag inside head tag by using attribute defer
<script src="/app.js" defer></script>